main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -237,6 +242,24 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       }
     }
 
+    def Effect: Rule1[ParsedAst.Declaration] = {
+      def Head = rule {
+        Documentation ~ Modifiers ~ SP ~ keyword("eff") ~ WS ~ Names.Effect ~ optWS ~ TypeParams
+      }
+
+      def EmptyBody = namedRule("EffectBody") {
+        push(Nil) ~ SP
+      }
+
+      def NonEmptyBody = namedRule("EffectBody") {
+        optWS ~ "{" ~ zeroOrMore(Declarations.Op) ~ optWS ~ "}" ~ SP
```

Comment:
Why space on the other side of `{`?

Reply:
Whitespace*

Reply:
This is a parallel to what's done in classes and instances. I'll try removing it and see what happens.